src/test/java/kafka/vertx/demo/WebSocketServerTest.java

```diff
@@ -0,0 +1,47 @@
+package kafka.vertx.demo;
```

Comment:
We're missing a copyright header at the top of here

Reply:
Added